### ISSUE TITLE
Quarto-Project under unc-path

### ISSUE THREAD

**Issue by Rathpete** [quarto-dev/quarto-cli#758]
Quarto-Project dosen´t work on an unc-path

**Comment by cderv** [comment]
Can you share more details about your environment, and also about the error message. 

Quarto is build using different tech stack, and this kind of UNC issue could come from some of them and not just within quarto code. For example, Pandoc had some issues in the past about UNC path and even if they improve, there could still be some problem. 

We need more information to know which part it comes from. Can you share a reproducible example for you ? Or at least some more information ? 

Thanks!

**Comment by Rathpete** [comment]
if i render the example qmd from an unc-path under Rstudio i get this information (pdf-fomat):
PDF.js Version 2.8.335 (build: 228adbf67)
Nachricht: Missing PDF "http://localhost:4910/web/abc.pdf".
Under html the message is: Internal server error
On a local path everything works fine...

**Comment by Rathpete** [comment]
Yes abc.pdf is corrctly created an i can open the file with an PDF viewer!
I use RStudio 2022.02.2
If I try to creat a new quarto-Project under RStudio the creation of the example .qmd fails for an project-subdirectory with an unc-path.

![grafik](https://user-images.githubusercontent.com/31278285/165950073-405dc3c5-e554-46db-a30c-7963c61fb822.png)

**Comment by cderv** [comment]
So this is definitely related to the RStudio IDE.  

There is an issue open already about UNC path on windows and the IDE with quarto : https://github.com/rstudio/rstudio/issues/10968 

Yours seems to be about the PDF viewer not being able to open the created file. @jjallaire does you know if this can be improved or it is a limit of the IDE for Quarto PDF support ?
